packages/block-editor/src/components/block-navigation/block.js

```diff
@@ -96,26 +96,24 @@ export default function BlockNavigationBlock( {
 				<>
 					<TreeGridCell className={ moverCellClassName }>
 						{ ( props ) => (
-							<BlockMoverUpButton
-								__experimentalOrientation="vertical"
-								clientIds={ [ clientId ] }
-								{ ...props }
-							/>
-						) }
-					</TreeGridCell>
-					<TreeGridCell className={ moverCellClassName }>
-						{ ( props ) => (
-							<BlockMoverDownButton
-								__experimentalOrientation="vertical"
-								clientIds={ [ clientId ] }
-								{ ...props }
-							/>
+							<div className="block-editor-block-navigation-block__mover-cell-alignment-wrapper">
+								<BlockMoverUpButton
+									__experimentalOrientation="vertical"
+									clientIds={ [ clientId ] }
+									{ ...props }
+								/>
+								<BlockMoverDownButton
+									__experimentalOrientation="vertical"
+									clientIds={ [ clientId ] }
+									{ ...props }
+								/>
+							</div>
```

Comment:
The `props` passed by TreeGridCell to the buttons are used to manage the tab index on focusable elements, so that the whole TreeGrid uses a roving tab index. Moving them both into the same cell so that they use the same props will result in that not working correctly (both buttons will get the same tabIndex).

I mentioned on this [PR](https://github.com/WordPress/gutenberg/pull/22590#discussion_r429753670) that it'd probably be better if the props were passed individually rather than spread as a single `...props` object—making it explicit would more clearly show the connection between the components. I think that's why this issue has happened. The block settings menu seems to need to be fixed too, I'll try to spin up a PR to handle that.

I'm not sure if it's a negative to have two focusable elements within a single cell in this tree grid pattern. If it's ok then it'd probably be easiest to have separate `TreeGridCell` and `TreeGridItem` components, where `TreeGridItem` doesn't have the wrapping `td`.

If not, then the other approach would probably be to use CSS grid. On https://github.com/WordPress/gutenberg/pull/22314/files#diff-3d2c7e20b234263753f1a83b39a15724R31-R33 I experimented with that and it works ok. Though it'd need some work for IE11. I don't think IE11 supports the named grid areas and some of the property names are different.

Reply:
Here's a PR that fixes the issue with the Block Settings Menu- https://github.com/WordPress/gutenberg/pull/22885.

Reply:
@talldan good notes! I just added `TreeGridItem` to this PR and it works quite well. The only part that's debatable is how left/right arrows should work when the focus is on these vertical movers. Should the right arrow move the focus from the top mover to the bottom mover? Or should that be the down arrow? Personally I think down arrow would make more sense here.

Reply:
@adamziel Yep, agreed. I made an issue for that and it's already had some discussion - https://github.com/WordPress/gutenberg/issues/22316.

It's an issue that exists in the main block toolbar as well. I don't think it's a blocker for this.

The conflict is that left/right makes sense for users who rely only on a screen reader (e.g. users who might have impaired vision), but it's definitely confusing for users using the visual layout of the buttons.